common/mount.py

```diff
@@ -57,9 +58,8 @@ def __init__(self, cfg = None, profile_id = None, tmp_mount = False, parent = No
                 cmd = [bit, 'pw-cache', action]
                 logger.debug('Call command: %s'
                              %' '.join(cmd), self)
-                proc = subprocess.Popen(cmd,
-                                        stdout = open(os.devnull, 'w'),
-                                        stderr = open(os.devnull, 'w'))
+                with open(os.devnull, 'w') as out:
```

Comment:
Rather use `subprocess.DEVNULL` which will do this automatically. BIT requires Python >= v3.3 anyways.